12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,44 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+  
+	ls -d [a-z]*
 
   2. All files whose name starts with an uppercase letter
   Answer:
 
+	ls -d [A-Z]*
+  
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
+	ls -d *[a,e,i,o,u]
+  
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
 
+	ls -d *[!a,e,i,o,u]
+  
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
+	
+	ls ./src/*.c
 
   6. All files with extension ".c" under any folder
   Answer:
 
+    find . -name '*.c'
+	ls -R | grep '.*[.]c'
```

Comment:
oh and as a side note I think that Ynon meant "in any folder *non-recursively* "... 
so ``` */*.c ``` would do the job.
 
but in the real world you'll want the recursive thing way more often :)